### ISSUE TITLE
Fail to support the saved model by Tensorflow 2.0

### ISSUE THREAD

**Issue by Hang-Zhang** [lutzroeder/netron#801]
We can use the following code to generate and save a simple model.

`import tensorflow as tf`
`model = tf.keras.Sequential([tf.keras.layers.Dense(5, input_shape=(3,)), tf.keras.layers.Softmax()])`
`model.save('model')`

However, when I open the saved .pb file through netron, the strange result is generated (as show below):
![Screen Shot 2021-09-05 at 10 31 43 PM](https://user-images.githubusercontent.com/32285641/132130911-f3af04d0-6d96-4b35-b02f-fff616780a36.png)

The attachment is the generate .pb file. Could you please solve the problem?
[model.zip](https://github.com/lutzroeder/netron/files/7111810/model.zip)
